### PR TITLE
Adding `IsInt` Converter

### PR DESCRIPTION
**Pull request by lazyplatypus**
**Description:** 
Added IsInt converter

**Link to tracking Issue:** #27894

**Testing:** <Describe what testing was performed and which tests were added.>
Added 

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Comment by codeboten** [comment]
THanks @lazyplatypus! Please sign the CLA

File: pkg/ottl/ottlfuncs/README.md [hunk 1]
```diff
@@ -542,6 +543,23 @@ Examples:
...
 
+### IsInt
+
+`IsInt(value)`
+
+The `IsInt` Converter returns true if the given value is a integer.
+
+The `value` is either a path expression to a telemetry field to retrieve or a literal. 
+
+If `value` is an `int64` or `pcommon.ValueTypeInt` then returns `true`, otherwise returns `false`.
```
Comment (strawgate) [inline pkg/ottl/ottlfuncs/README.md:554]: Looks like int8, etc also result in a true return

Comment (TylerHelmuth) [reply pkg/ottl/ottlfuncs/README.md:554]: Although OTTL basically forces int64, its true that the function is less strict, so we can make the README reflect reality like @strawgate mentions.

**Review by TylerHelmuth — COMMENTED** [review]
@lazyplatypus @danielnewrelic it looks like at least 1 commit in the PR is associated to an email that has not signed the CLA.

**Pull request closed without merging**